### PR TITLE
chore: add Realtime Postgres Changes guide

### PR DESCRIPTION
**Pull request by w3b6x9**

### CODE REVIEW

File: apps/reference/docs/guides/realtime/postgres_changes.mdx [hunk 1]
```diff
@@ -0,0 +1,83 @@
+---
+id: postgres_changes
```
Comment (inian) [inline apps/reference/docs/guides/realtime/postgres_changes.mdx:2]:
```suggestion
id: postgres-changes
```

File: apps/reference/docs/guides/realtime/postgres_changes.mdx [hunk 2]
```diff
@@ -0,0 +1,83 @@
+---
```
Comment (inian) [inline apps/reference/docs/guides/realtime/postgres_changes.mdx:1]: Lets change the file name to postgres-changes.mdx

File: apps/reference/nav/_referenceSidebars.js [hunk 3]
```diff
@@ -84,7 +84,11 @@ const sidebars = {
       type: 'category',
       label: 'Realtime',
       collapsed: true,
-      items: ['guides/realtime/broadcast', 'guides/realtime/presence'],
+      items: [
+        'guides/realtime/broadcast',
+        'guides/realtime/presence',
+        'guides/realtime/postgres_changes'
```
Comment (inian) [inline apps/reference/nav/_referenceSidebars.js:90]:
```suggestion
        'guides/realtime/postgres-changes'
```

**Review by inian — APPROVED** [review]

File: apps/reference/docs/guides/realtime/postgres_changes.mdx (hunk 2 above)
Comment (w3b6x9) [reply apps/reference/docs/guides/realtime/postgres_changes.mdx]: done

**Pull request merged**